Derpitcraft/README.md

```diff
@@ -171,18 +170,19 @@ You can't get an inifinte number of slime chunks. The limits imposed are as foll
 - `/trigger nc_inNether`: Converts current coordinates to overworld coordinates. Can be used as a way to efficiently calculate coordinates for portals.
 - `/trigger nc_inOverworld`: Converts current coordinates to nether coordinates.Can be used as a way to efficiently calculate coordinates for portals.
 - `/aboutdc`: Shows information about Derpitcraft.
+- `/tm | /tmsg | /teammsg`: Message within your team.
 
-### Using Proximity chat (Super Voice Chat)
+### Using proximity chat (Simple Voice Chat)
 
-Derpitcraft has a proximity chat system powered by the *simple voice chat* mod. You can download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.
+Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. You can download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.
 
 Once you have downloaded the mod, it should work out of the box. Read more about the mod on it's wiki (https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.
 
 You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard.
 
-### DCCoordinates and Claims
+### Coordinates and Claims
```

Comment:
This section is specifically about the DCCoordinates command and not Coordinates in general. This is ambiguous.

Reply:
made it explicit